corehq/ex-submodules/phonelog/utils.py

```diff
@@ -191,11 +191,14 @@ def __init__(self, domain, xform):
         self.xform = xform
 
     def send_data(self, url):
-        send_device_log_to_sumologic.delay(url, self.log_subreport(), self._get_header('log'))
-        send_device_log_to_sumologic.delay(url, self.user_error_subreport(), self._get_header('user_error'))
-        send_device_log_to_sumologic.delay(url, self.force_close_subreport(), self._get_header('force_close'))
-
-    def _get_header(self, fmt):
+        send_device_log_to_sumologic.delay(url, self.log_subreport(),
+                                           self._get_header('log', False))
+        send_device_log_to_sumologic.delay(url, self.user_error_subreport(),
+                                           self._get_header('user_error', False))
+        send_device_log_to_sumologic.delay(url, self.force_close_subreport(),
+                                           self._get_header('force_close', False))
+
+    def _get_header(self, fmt, should_encode=True):
```

Comment:
Does this method still need the `should_encode` parameter?

Reply:
nope, missed that one when taking out the others.